core/frontend/tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
   "compilerOptions": {
-    "outDir": "./lib"
+    "esModuleInterop": true,
```

Comment:
Why is this needed?

Reply:
There were some imports where we used require, which prevented us from delivering an esm variant. Added this which provides a thin wrapper in the transpiled code to let us import these cjs pkgs: https://www.typescriptlang.org/tsconfig#esModuleInterop

Reply:
okay, sounds good.